css/RTstyle.css

```diff
@@ -16,6 +16,7 @@
 
 /* Modal Content */
 .modal-content {
+    
```

Comment:
Why do you add empty space inside the CSS class definition?
In general there are many redundant spaces in this file which hinder readability.